armstrongNumber/armstrongNumberProgram.java

```diff
@@ -0,0 +1,24 @@
+package armstrongNumber;
+
+public class armstrongNumberProgram {
```

Comment:
Class name should start with capital letter

Reply:
Done changes